src/python/test/test_dx_app_wizard_and_run_app_locally.py

```diff
@@ -205,6 +205,19 @@ def test_dx_run_app_locally_and_compare_results(self):
         result = remote_job.describe()
         self.assertEqual(result["output"]["out1"], 140)
 
+    def test_dx_verify_build_app(self):
+        appdir = create_app_dir()
+        print("Setting current project to", self.project)
+        dxpy.WORKSPACE_ID = self.project
+        dxpy.PROJECT_CONTEXT_ID = self.project
+        bundled_resources = dxpy.app_builder.upload_resources(appdir)
+        applet_id, _ignored_applet_spec = dxpy.app_builder.upload_applet(appdir, bundled_resources, overwrite=True, dx_toolkit_autodep=False)
+        app_obj = dxpy.DXApplet(applet_id)
+        try:
+            app_obj.describe()
+        except dxpy.exceptions.ResourceNotFound as dxrne:
+            self.fail(dxrne.error_message())
+
```

Comment:
It would be useful to have a test name that describes better what exactly is being tested test, for example test_upload_applet_with_current_context_project (or sth similar).

Since we expect the build to succeed, we can add an assert, for example:
`self.assertEquals(app_obj.describe()['id'], app_obj.get_id())`
or
`self.assertEquals(app_obj.describe()['project'], dxpy.WORKSPACE_ID)`